### PR TITLE
5 lighthouse action

### LINKED ISSUE #5: GitHub Action for Lighthouse
Set up Google Lighthouse as a GH Action so we can monitor a11y and SEO

https://developer.chrome.com/docs/lighthouse/overview/

### PR DESCRIPTION
**Pull request by RossellaFer**
Resolves #5 Github action for Lighthouse

## What changed 🧐
- setup the action [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci/tree/main?tab=readme-ov-file)
The action will test the homepage 3 times and generate a report with the scores, similar to this [this one](https://github.com/WomenInSoftwareEngineeringJP/home/actions/runs/8830675322/job/24244314068))
- changed the ESLint configuration to work with the `lighthouserc.cjs` file

## How did you test it? 🧪

Manual testing by running the commands

```
npm install && npm install -g @lhci/cli@0.13.x
npm run build
lhci autorun
```

### CODE REVIEW

File: lighthouserc.cjs [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      },
+      collect: {
+        staticDistDir: './dist',
+        "url": [
+          "http://localhost/"
+        ]
+      },
+      asserts: {
+        assertions: {
+          'categories:accessibility': ['warn', { minScore: 1 }],
```
Comment (ann-kilzer) [inline lighthouserc.cjs:14]: nit: Can we make the syntax similar, e.g. either use `1` or `1.0` ?

File: lighthouserc.cjs [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+          'categories:best-practices': ['warn', { minScore: 1.0 }],
+          'categories:performance': ['warn', { minScore: 0.2 }],
```
Comment (ann-kilzer) [inline lighthouserc.cjs:16]: Was our performance that bad 🤣

File: lighthouserc.cjs (hunk 1 above)
Comment (ann-kilzer) [reply lighthouserc.cjs:14]: TBH I think aiming for 90% or better is probably a healthier metric for the MVP phase. Usually 100% becomes an impossible standard

File: lighthouserc.cjs (hunk 2 above)
Comment (RossellaFer) [reply lighthouserc.cjs:16]: I think 0.9 is a good benchmark for SEO and accessibility, while 0.8 is better for best-practices and SEO. What do you think?

Comment (ann-kilzer) [reply lighthouserc.cjs]: Sounds good!

**Review by ann-kilzer — APPROVED** [review]
lgtm

**Pull request merged**